GOVERNANCE.md

```diff
@@ -75,10 +75,10 @@ The current list of Infra members is as below, in alphabetical order on last nam
 
 | Name                                  | GitHub                                           |
 | ------------------------------------- | :----------------------------------------------- |
-| Zili Chen <wander4096@gmail.com>      | [tisonkun](https://github.com/tisonkun)          |
+| Junchen Ye <yejunchen@pingcap.cn>     | [VelocityLight](https://github.com/VelocityLight)|
```

Comment:
Emmm .. I'm still actively oversee the org. Why remove without any communication or even notification?